### DISCUSSION TITLE
checkDebugDuplicateClasses FAILED

### DISCUSSION THREAD

**Discussion by FakriSouyo** [community/community#102504] [Programming Help] [answered]
can someone help me with this problem,

![Screenshot (153)](https://github.com/community/community/assets/63139619/936f9a55-3f85-49cf-9644-99481759ba06)

This my Build.gradle(Module :app)

    plugins {
        id("com.android.application")
        id("org.jetbrains.kotlin.android")
    }

    android {
        namespace = "com.fakri.skripsi"
        compileSdk = 34

    defaultConfig {
        applicationId = "com.fakri.skripsi"
        minSdk = 31
        targetSdk = 33
        versionCode = 1
        versionName = "1.0"

        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
        vectorDrawables {
            useSupportLibrary = true
        }
    }

    buildTypes {
        release {
            isMinifyEnabled = false
            proguardFiles(
                getDefaultProguardFile("proguard-android-optimize.txt"),
                "proguard-rules.pro"
            )
        }
    }
    compileOptions {
        sourceCompatibility = JavaVersion.VERSION_1_8
        targetCompatibility = JavaVersion.VERSION_1_8
    }
    kotlinOptions {
        jvmTarget = "1.8"
    }
    buildFeatures {
        compose = true
    }
    composeOptions {
        kotlinCompilerExtensionVersion = "1.4.3"
    }
    packaging {
        resources {
            excludes += "/META-INF/{AL2.0,LGPL2.1}"
        }
    }

    buildFeatures {
        viewBinding = true
        mlModelBinding = true
    }

    }

    dependencies {

    implementation("androidx.core:core-ktx:1.12.0")
    implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.8.0-alpha01")
    implementation("androidx.activity:activity-compose:1.9.0-alpha02")
    implementation("androidx.activity:activity-ktx:1.9.0-alpha02")
    implementation(platform("androidx.compose:compose-bom:2023.03.00"))
    implementation("androidx.compose.ui:ui")
    implementation("androidx.compose.ui:ui-graphics")
    implementation("androidx.compose.ui:ui-tooling-preview")
    implementation("androidx.compose.material3:material3")
    implementation("androidx.wear.compose:compose-material3:1.0.0-alpha16")
    implementation("androidx.room:room-compiler:2.6.1")
    implementation("org.tensorflow:tensorflow-lite-support:0.4.4")
    implementation("org.tensorflow:tensorflow-lite-metadata:0.4.4")
    testImplementation("junit:junit:4.13.2")
    androidTestImplementation("androidx.test.ext:junit:1.1.5")
    androidTestImplementation("androidx.test.espresso:espresso-core:3.5.1")
    androidTestImplementation(platform("androidx.compose:compose-bom:2023.03.00"))
    androidTestImplementation("androidx.compose.ui:ui-test-junit4")
    debugImplementation("androidx.compose.ui:ui-tooling")
    debugImplementation("androidx.compose.ui:ui-test-manifest")
    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.7.1")
    implementation("androidx.core:core-splashscreen:1.0.1")
    implementation("androidx.navigation:navigation-compose:2.7.6")
    implementation ("io.coil-kt:coil-compose:2.4.0")
    implementation("androidx.appcompat:appcompat:1.6.1")
    implementation ("androidx.camera:camera-camera2:1.3.1")
    implementation ("androidx.camera:camera-lifecycle:1.3.1")
    implementation("androidx.fragment:fragment-ktx:1.7.0-alpha09")
    implementation("org.tensorflow:tensorflow-lite-support:0.4.4")

    // https://mvnrepository.com/artifact/com.google.accompanist/accompanist-coil
    implementation("com.google.accompanist:accompanist-coil:0.15.0")

    }

This for Build.gradle (Project)

       plugins {
           id("com.android.application") version "8.1.1" apply false
           id("org.jetbrains.kotlin.android") version "1.8.10" apply false
      }

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by heysem-useinsider** [comment, marked as answer]
Hi, I hope you've found the answer by now.

In any case, I noticed that you have the dependency `org.tensorflow:tensorflow-lite-support` twice in your Gradle file. You should probably remove one of them:

```diff
-    implementation("org.tensorflow:tensorflow-lite-support:0.4.4")
```

If that doesn't work please run the command `./gradlew app:dependencies` in your project and post the output here so I can help more.

**Reply by FakriSouyo** [reply]
Thanks
